workspaces/cogito-encryption/package.json

```diff
@@ -39,7 +39,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@cogitojs/crypto": "0.2.x",
```

Comment:
Too bad that this is not supported by lerna..

Reply:
Not supported? I thought it works...